settings.py

```diff
@@ -1,19 +1,32 @@
 import os
 import dj_database_url  # type: ignore
+import boto3
+
+# Get db paasword from AWS SSM
+session = boto3.Session(profile_name="default")
+ssm = session.client("ssm")
+
+parameter = ssm.get_parameter(
+    Name=os.environ.get("STAGE_PARAM_NAME"), WithDecryption=True
```

Comment:
Why `STAGE_PARAM_NAME`? Is that a placeholder for later?

Reply:
it's not a place holder for later, it's to ensure we don't hardcode the param for stage dev or stage prod.  @johnseekins